core/util_test.go

```diff
@@ -285,13 +302,20 @@ func (d *DrandTestScenario) RunDKG() *key.Group {
 			group, err := key.GroupFromProto(groupPacket)
 			require.NoError(d.t, err)
 
-			d.t.Logf("[RunDKG] NonLeader Finished. GroupHash %x", group.Hash())
+			d.t.Logf("[RunDKG] NonLeader %s Finished. GroupHash %x", node.GetAddr(), group.Hash())
+
+			// We need to make sure the daemon is running before continuing
+			d.waitRunning(client, n)
 			wg.Done()
 		}(node)
 	}
 
+	d.CheckStatuses(d.nodes)
 	// wait for all to return
 	wg.Wait()
+
+	d.CheckStatuses(d.nodes)
```

Comment:
Why is this here ?

Reply:
This is some extra logs to try and see if the Sleeps are really used in the CI or not.